openedx/core/djangoapps/user_api/tests/test_models.py

```diff
@@ -77,6 +77,18 @@ def test_create_user_org_tags(self):
         self.assertEquals(tag.value, "barfoo")
         self.assertNotEqual(original_modified, tag.modified)
 
+    def test_retire_user_org_tags_by_user_value(self):
```

Comment:
It might be worth adding a test where two users point to the same org. The test would show that a delete by user doesn't delete all the org's entries.

Reply:
Great call. I just added that. Now if only those mystery failures would stop...